doc: Update the grafana docs with the new grafana integration settings (#9074)
solo-io/gloo: docs/content/guides/observability/grafana/dashboards/_index.md

1. [Whole-Cluster Dashboard](#whole-cluster-dashboard)
1. [Dynamically Generated Dashboards](#dynamically-generated-dashboards)

> 1. [Custom Prefix for Dashboard UIDs and Titles](#custom-prefix-for-dashboard-uids-and-titles)

Reviewer: ```suggestion
1. [Custom prefix for dashboard UIDs and titles](#custom-prefix-for-dashboard-uids-and-titles)
```

* `.NameTemplate`
* `.Uid`
* `.EnvoyClusterName`
* `.DashboardPrefix`
* `.ExtraMetricQueryParameters`

> ## Custom Prefix for Dashboard UIDs and Titles

Reviewer: ```suggestion
## Custom prefix for dashboard UIDs and titles
```

> Users can provide a prefix that applies to the UID and Title of all dashboards generated by Gloo Edge Enterprise. This is extremely useful in scenarios where there are multiple installations of Edge Enterprise that publish dashboards to an external centralized Grafana Server.

Reviewer: ```suggestion
You can provide a prefix that applies to the UID and title of all dashboards that are generated by Gloo Edge Enterprise. This prefix can be very useful in scenarios where you have multiple installations of Edge Enterprise that publish dashboards to an external centralized Grafana Server.
```

Providing a unique prefix per installation ensures that all dashboards generated by Gloo Edge Enterprise have a unique UID and Title and do not conflict across multiple installations.

> This can be done by specifying the `gloo.settings.observabilityOptions.grafanaIntegration.dashboardPrefix` setting with the desired prefix.

Reviewer: ```suggestion
To enable this feature, add the prefix that you want to use in the `gloo.settings.observabilityOptions.grafanaIntegration.dashboardPrefix` field. 
```

> **Note**: This prefix is restricted to 20 characters in length

## Extra Metric Query Parameters

> Gloo Edge Enterprise's observability component also gives users the ability to provide extra query parameters to add to all metric query definitions in the generated Grafana Dashboards. A good use case is when metrics from multiple clusters are aggregated to a unified prometheus server. The user can define the extra metric query parameters with the desired cluster name such as `cluster="some-cluster"`. This ensures that metric definitions in dashboards generated by the given Edge installation will contain this additional query parameter and thereby only fetch metrics matching the cluster with the given name.

Reviewer: ```suggestion
You can specify extra query parameters that you want to add to all metric query definitions in the Grafana dashboards that Gloo Edge generates. For example, you might aggregate metrics from multiple clusters in a central Prometheus server. Without additional query parameters, the Gloo Edge-generated dashboards pull in the metrics for all of your clusters which can lead to incorrect results and graphs. To ensure that the Gloo Edge Grafana dashboards show only the metrics for the cluster that you are interested in, you can specify the cluster name as an additional query parameter, such as `cluster="my-cluster"`. Only metrics that match this parameter are pulled into the Grafana dashboard. 
```

> This can be done by specifying the `gloo.settings.observabilityOptions.grafanaIntegration.extraMetricQueryParameters` setting with the desired query parameters. It can consist of multiple query parameters separated by a comma. For example `cluster="some-cluster",gateway_proxy_id="proxy-2"`.

Reviewer: ```suggestion
To enable this feature, add the query parameters that you want to add to the `gloo.settings.observabilityOptions.grafanaIntegration.extraMetricQueryParameters` field. Separate multiple query parameters by comma, such as `cluster="my-cluster",gateway_proxy_id="proxy-2"`.
```
